api/db/changelog/db.changelog-153-cdr-version-addDefaultValue-data_access_level.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="thibault" id="db.changelog-153-cdr-version-addDefaultValue-data_access_level">
+    <!--
+    Set default value to REGISTERED (1) as a step in the deletion of this field.
+    After removing DbCdrVersion accessors, Hibernate complains about the lack of a default value here.
+    -->
+    <addDefaultValue tableName="cdr_version" columnName="data_access_level" defaultValue="1"/>
```

Comment:
See #4635 for the same change for DbWorkspace

Reply:
REGISTERED is correct for every CDR Version in Staging/Stable/Perf/Preprod/Prod, and nearly all versions in test/local.  We should be able to delete this column before it is ever untrue in higher environments.

Reply:
Should this column be dropped from the table since property dataAccessLevel is no longer accessible from DbCdrVersion model?

Reply:
Yes.  I intend to do so in the near future: https://precisionmedicineinitiative.atlassian.net/browse/RW-6189

I'm not doing so in this PR as a conservative step to help with possible problems discovered after merge, release version contention, ease of rollback, etc.